script.js

```diff
@@ -0,0 +1,5 @@
+let width
+let height
+let area = width * height
+
+console.log(area)
```

Comment:
why not `const`? you do not modify it thus you should use const (it should be your "go-to" choice)